lib/mongo/collection/view/writable.rb

```diff
@@ -55,6 +56,10 @@ def find_one_and_delete
         # @option opts [ true, false ] :upsert Whether to upsert if the document doesn't exist.
         # @option opts [ true, false ] :bypass_document_validation Whether or
         #   not to skip document level validation.
+        # @option opts [ true, false ] :upsert Whether to upsert if the
+        #   document doesn't exist.
```

Comment:
Oh that's weird, I rebased against master before comitting. I'll get rid of that line, I'm not sure what happened.